mbed_flasher/flashers/enhancedserial.py

```diff
@@ -11,12 +11,16 @@
 like found in the telnetlib.
 """
 import re
+import termios
+from time import sleep
 import pkg_resources
 from serial import Serial, SerialException, SerialTimeoutException
-from time import sleep
 
 
-class EnhancedSerial(Serial):
+class EnhancedSerial(Serial): # pylint: disable=too-many-ancestors
```

Comment:
This is also class that could be common python lib repository, @juhhov @jonikula